src/Agent/NewRelic/Profiler/Profiler/EntryPoint.cpp

```diff
@@ -20,14 +20,13 @@ BOOL STDMETHODCALLTYPE DllMain(HMODULE hModule, DWORD ul_reason_for_call, LPVOID
     return TRUE;
 }
 
-extern "C" {
-    HRESULT STDMETHODCALLTYPE DllGetClassObject(REFCLSID rclsid, REFIID riid, void **ppv)
+extern "C"
```

Comment:
This comment is not for this exact line of code. Since the profiler no longer depends on the `CLSID_NewRelicProfiler` and `CLSID_NewRelicCorCLRProfiler`, should we remove these ?

Reply:
Removed them as suggested.